### DISCUSSION TITLE
关于定时购票

### DISCUSSION THREAD

**Discussion by Coin-Lu** [mikumifa/biliTickerBuy#97] [提问专区]
流程是否可以是提前十分钟设定好抢票时间后点击开始抢票，期间保持界面不关闭？

**Comment by mikumifa** [comment]
最好这么做，然后可以提前几分钟定

**Reply by Coin-Lu** [reply]
假设我购买的票8:00开售，提前几分钟的意思是我设定时间就将它设定到7:58，还是将设定时间为8:00，7:58点击“开始抢票”？

**Reply by mikumifa** [reply]
差不多，提前一点好，这个计时器有一点延迟

**Reply by Colin-Chen630** [reply]
> 差不多，提前一点好，这个计时器有一点延迟

所以抢票开始时间设置为开票前一两分钟不会触发过早抢票的封号机制是吗？
